ddd-koans/Value Object/Tiny Types/test/Tests.kt

```diff
@@ -0,0 +1,142 @@
+import org.assertj.core.api.Assertions.*
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+import org.junit.jupiter.params.provider.ValueSource
+import kotlin.reflect.KMutableProperty
+import kotlin.reflect.KVisibility
+import kotlin.reflect.full.declaredMemberProperties
+import kotlin.reflect.jvm.jvmErasure
+
+class Test {
+
+    @ParameterizedTest
+    @ValueSource(strings = ["Numerator", "Denominator"])
+    fun testIfNumeratorAndDenominatorClassAreAvailableInClassPath(className: String) {
+        assertThatCode { Class.forName(className) }
```

Comment:
is it possible to assert if those classes are defined as `data classes`?

Reply:
Yes, it is. I will add tests for the case.